filebeat/harvester/reader/multiline.go

```diff
@@ -225,6 +227,8 @@ func (mlr *Multiline) clear() {
 // finalize writes the existing content into the returned message and resets all reader variables.
 func (mlr *Multiline) finalize() Message {
 
+	lines := common.MapStr{"lines": mlr.numLines}
+	mlr.message.AddFields(common.MapStr{"multiline": lines})
```

Comment:
What will happen if someone defines `json` or `multiline` as fields in the config file, together with `fields_under_root`?